sdk/webpubsub/azure-messaging-webpubsubservice/README.md

```diff
@@ -59,22 +91,16 @@ In order to interact with the Azure WebPubSub service, you'll need to create an
 
 ```python
 >>> from azure.messaging.webpubsubservice import WebPubSubServiceClient
->>> from azure.core.credentials import AzureKeyCredential
->>> from azure.messaging.webpubsubservice.rest import build_send_to_all_request
->>> client = WebPubSubServiceClient(endpoint='<endpoint>', credential=AzureKeyCredential('somesecret'))
->>> request = build_send_to_all_request('default', json={ 'Hello':  'webpubsub!' })
->>> request
-<HttpRequest [POST], url: '/api/hubs/default/:send?api-version=2020-10-01'>
->>> response = client.send_request(request)
->>> response
-<RequestsTransportResponse: 202 Accepted>
->>> response.status_code
-202
+>>> from azure.identity import DefaultAzureCredential
+>>> from azure.core.exceptions import HttpResponseError
+
+>>> client = WebPubSubServiceClient(endpoint='<endpoint>', credential=DefaultAzureCredential())
 >>> with open('file.json', 'r') as f:
->>>    request = build_send_to_all_request('ahub', content=f, content_type='application/json')
->>>    response = client.send_request(request)
->>> print(response)
-<RequestsTransportResponse: 202 Accepted>
+    try:
+        client.web_pub_sub.send_to_all('ahub', content=f, content_type='application/json')
```

Comment:
is it possible to get rid of this `web_pub_sub`? since the client is already `WebPubSubServiceClient`, `web_pub_sub` sounds redundant.

Reply:
`web_pub_sub` is operation_group of `WebPubSubServiceClient` just like `health_api`. Through operation_group, we could define same name for function of different opearion_group. For example : client.web_pub_sub.send_to_all or client.web_pub_sub_2.send_to_all

Reply:
removed `health_api` and moved everything under `web_pub_sub` directly onto the client